instrumentation/kafka/kafka-clients/kafka-clients-2.6/library/src/main/java/io/opentelemetry/instrumentation/kafkaclients/TracingConsumerInterceptor.java

```diff
@@ -34,5 +34,7 @@ public void onCommit(Map<TopicPartition, OffsetAndMetadata> offsets) {}
   public void close() {}
 
   @Override
-  public void configure(Map<String, ?> configs) {}
+  public void configure(Map<String, ?> configs) {
+    // TODO: support experimental attributes config
```

Comment:
Later, in a separate PR, we should refactor this classes so that they use a similar way to configure themselves as introduced in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/6138 -- it seems to be the "idiomatic" approach for kafka extensions, I guess